upup/pkg/fi/cloudup/awsup/machine_types.go

```diff
@@ -55,7 +55,7 @@ func (m *AWSMachineTypeInfo) EphemeralDevices() []*EphemeralDevice {
 			// TODO: What drive letters do we use?
 			klog.Fatalf("ephemeral devices for > 20 not yet implemented")
 		}
-		d.DeviceName = "/dev/sd" + string('c'+i)
```

Comment:
why this change is needed for openstack?

Reply:
It's not needed & I can revert it - without this change, the `govet` test would fail. Should I remove this commit?

Reply:
@zetaab this is needed because of changes in Go. I created a cherrypick for the fix https://github.com/kubernetes/kops/pull/9752.